src/main/resources/db/migration/general/V7_0__remove_old_tables.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE "UserNotificationSetting" CASCADE;
+
+DROP TABLE "ShiftNotification" CASCADE;
+
+DROP TABLE "ShiftTaskNotification" CASCADE;
+
+DROP TABLE "Prison" CASCADE;
+
+DROP TABLE "NotificationConfiguration" CASCADE;
+
+UPDATE shift_notification SET shift_type = 'shift' WHERE shift_type = 'SHIFT';
```

Comment:
Can we move these to a separate migration. It doesn't do what it says on the tin

Reply:
Good shout.